class.jetpack-gutenberg.php

```diff
@@ -309,5 +309,8 @@ public static function enqueue_block_editor_assets() {
 		Jetpack::setup_wp_i18n_locale_data();
 
 		wp_enqueue_style( 'jetpack-blocks-editor', $editor_style, array(), $version );
+
+		// The social-logos styles are used for Publicize service icons
```

Comment:
Can you add a TODO comment to roughly reflect

> Ideally, we'd want to ship the icons with the build [...]

Reply:
👍 https://github.com/Automattic/jetpack/pull/10653/commits/cf28ef0d2f42efbb965b0d25cf938c554fb576e1